src/table/components/virtualized-table/utils/get-height.ts

```diff
@@ -0,0 +1,11 @@
+import { COMMON_CELL_STYLING } from '../../../styling-defaults';
+import { GeneratedStyling } from '../../../types';
+import { fontSizeToRowHeight } from '../../../utils/styling-utils';
+import { MIN_BODY_ROW_HEIGHT, MIN_HEADER_HEIGHT } from '../constants';
+import { BodyStyle } from '../types';
+
+export const getHeaderRowHeight = ({ fontSize = COMMON_CELL_STYLING.fontSize }: GeneratedStyling) =>
+  Math.max(MIN_HEADER_HEIGHT, Math.round(fontSizeToRowHeight(fontSize)));
```

Comment:
why rounding?

Reply:
To fix an issue where the cell borders would sometimes slight overlap, making some borders appear bigger than others. My guess is that it's caused by a floating point precision issue as I went away when I added the `Math.round`

Reply:
I see. I was going to say you should test with really small font sizes, I think we support down to 5px. But now I see that you set a min height. Not sure if we want to add that limit, but also anything smaller than 24 this is tiny

Reply:
let's keep it this way until we know there is a user need for smaller rows